packages/parcel-runtime/src/utils/hmr-check.ts

```diff
@@ -99,11 +102,16 @@ function hmrAcceptCheckOne(
 
   hmrState.checkedAssets[id] = true
 
-  var cached = bundle.cache[id]
+  const cached = bundle.cache[id]
+
+  hmrState.assetsToDispose.push([bundle, id])
 
-  hmrState.assetsToAccept.push([bundle, id])
+  if (!cached || (cached.hot && cached.hot._acceptCallbacks.length)) {
```

Comment:
I feel like you should get rid of the underscore in `acceptCallbacks`. It's used all over the place and not really semantically private.

Reply:
FYI that's required to match up with parcel's module, I wouldn't use underscore either lol